waku/factory/internal_config.nim

```diff
@@ -32,7 +32,14 @@ proc enrConfiguration*(
   let shards: seq[uint16] =
     # no shards configured
     if conf.shards.len == 0:
-      toSeq(0 ..< conf.pubsubTopics.len).mapIt(uint16(it))
+      var shardsLocal = newSeq[uint16]()
+      let shardsRes = topicsToRelayShards(conf.pubsubTopics)
+      if shardsRes.isOk() and shardsRes.get().isSome():
+        shardsLocal = shardsRes.get().get().shardIds
+      else:
+        error "failed to parse pubsubTopic", error = shardsRes.error
```

Comment:
I would suggest error message to be actionable statement.

egg. Parsing pubsub topic failed. Please format according to static shard specification.